styles/base/form.scss

```diff
@@ -41,8 +41,54 @@
   &.-sun {
     @include form-variant($brand-sun);
   }
+
+  &.-error {
+    @include form-variant($brand-crimson);
+    background-color: lighten($brand-crimson, 30);
+    color: $brand-crimson-dk;
+    &::-webkit-input-placeholder {
+      color: $brand-crimson-dk;
+    }
+  }
+}
+
+.form-group {
+  position: relative;
 }
 
+.form-group > .message {
+  position: absolute;
+  top: 40px;
+  left: 20px;
+  z-index: 9999;
+  visibility: hidden;
+  width: 220px;
+  text-align: center;
+  color: #FFFFFF;
+  line-height: 2;
```

Comment:
I think we'll have messages that will use multiple lines, so let's remove this rule.